### PR TITLE
Updated grunt and readme to reflect current run perspective and usage…

### PR DESCRIPTION
**Pull request by paullryan**
… and fixed module load capitalization.

Let me know if you have issues with this.

### CODE REVIEW

**Comment by tivie** [comment]
Thanks @paullryan . I will take a look as soon as possible.

**Comment by tivie** [comment]
https://github.com/showdownjs/showdown#contributing

File: Gruntfile.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = function (grunt) {
       },
       dist: {
         src:  [
-          'src/*/**.js'
+          'src/**/**.js'
```
Comment (posva) [inline Gruntfile.js]: it should be `**/*.js`

Comment (acourtiol) [inline Gruntfile.js]: :+1:

**Comment by SyntaxRules** [comment]
Solved with #1. Thanks for your time in doing this for us! I'd be happy to include your example in the `readme.md` if you can pull that out into its own pull request.

**Pull request closed without merging**